src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -120,6 +144,23 @@ CAC.Map.Control = (function ($, L, _) {
     }
 
     /**
+     * Show/hide sidebar options based on the selected mode.
+     * Expects both tabs to have the same selector names for the toggleable divs.
+     */
+    function changeMode(selectors) {
```

Comment:
Or better yet, if it needs to be on both sidebars, add a new BikeOptionsControl that encapsulates all the bike options logic. Then the bike triangle definition can come off the map control as well, since that seems like a weird spot for that as well.